src/plugins/vuetify.js

```diff
@@ -1,5 +1,6 @@
 import * as components from 'vuetify/components';
 import * as directives from 'vuetify/directives';
+import { VSkeletonLoader } from 'vuetify/labs/components';
```

Comment:
I didn't actually realize `v-skeleton-loader` was an unstable component when writing up issue #484, so this is a little bit sketchy but I think it is okay as the skeleton loader is being officially released in the upcoming 3.4.0 (Blackguard) Vuetify release.

@seriouslysean commenting to let you know I goofed on this one but I think it's still an improvement so I've written 502 to track upgrading and removing the Vuetify labs import once it releases